hoomd/hpmc/UpdaterRemoveDrift.h

```diff
@@ -43,10 +42,46 @@ template<class Shape> class RemoveDriftUpdater : public Updater
     public:
     //! Constructor
     RemoveDriftUpdater(std::shared_ptr<SystemDefinition> sysdef,
-                       std::shared_ptr<ExternalFieldLattice<Shape>> externalLattice,
-                       std::shared_ptr<IntegratorHPMCMono<Shape>> mc)
-        : Updater(sysdef), m_externalLattice(externalLattice), m_mc(mc)
+                       std::shared_ptr<IntegratorHPMCMono<Shape>> mc,
+                       pybind11::list ref_positions)
+        : Updater(sysdef), m_mc(mc)
         {
+        m_ref_positions.resize(m_pdata->getN());
+        setRefPositions(ref_positions);
+        }
+
+    //! Get reference positions as a python list of 3-tuples
+    pybind11::list getRefPositions()
```

Comment:
Given that this is `N` particles long. I think accepting a NumPy array would be preferable here, as that is typically how we store large numerical data (roughly the size of the simulation) versus smaller amounts of data like a shape definition or pair parameter parametrization.

Reply:
Makes sense but I'm not sure how to handle Numpy arrays with ParameterDict. Is this done in some other place that I can take as an example?

Reply:
No, I don't believe we have had an example of this yet, but I can whip up a type validator for `numpy.ndarray` fairly easily, and will ping you when done.

Reply:
Great, thanks.